images/macos/toolsets/Readme.md

```diff
@@ -6,8 +6,18 @@
   - `link` property points to the place where Xcode will be located on image. `/Applications/Xcode_<link>.app`  
   - `version` points to Xcode version that will be downloaded and installed
   - `symlinks` describes the list of aliases where symlinks will be created to
-  - `install_runtimes` is an array or boolean function to control over the related simulator runtimes, set of possible values: [ `iOS`, `watchOS`, `visionOS`, `tvOS` ], use `true` if you want to install all runtimes, use `false` if you want to skip runtimes installation
   - `sha256` used to check integrity of the Xcode installer file
+  - `install_runtimes` – controls the installation of simulator runtimes:
+    - `"all"` – installs all default runtimes.
+    - `false` – skips runtime installation.
+    - **Hashtable** – allows manual selection:
+      - Mandatory keys: `[ "iOS", "watchOS", "tvOS" ]`, plus `visionOS` for arm64 images.
+      - Values [string]:  
+        - `"default"` – installs the default runtime.  
+        - `"skip"` – skips installation.  
```

Comment:
nit: Would simply omitting the runtime have the same meaning as "skip"? Just trying to think if we can avoid introducing entities that may add to cognitive load with a small ROI

Reply:
In general, of course, it is possible. Why not. I just didn't want us to forget something one day. At least the script will remind us.